workspace-manager-clienttests/src/main/java/scripts/utils/WorkspaceManagerServiceUtils.java

```diff
@@ -0,0 +1,92 @@
+package scripts.utils;
+
+import bio.terra.testrunner.common.utils.AuthenticationUtils;
+import bio.terra.testrunner.runner.config.ServerSpecification;
+import bio.terra.testrunner.runner.config.TestUserSpecification;
+import bio.terra.workspace.client.ApiClient;
+import com.google.auth.oauth2.AccessToken;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.common.base.Strings;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+
+public class WorkspaceManagerServiceUtils {
+    private static final Logger logger = LoggerFactory.getLogger(WorkspaceManagerServiceUtils.class);
+
+    private WorkspaceManagerServiceUtils() {}
+
+    /**
+     * Build the Workspace Manager Service API client object for the given server specification.
+     *
+     * @param server the server we are testing against
+     * @return the API client object
+     */
+    public static ApiClient getClient(ServerSpecification server) throws IOException {
+        if (Strings.isNullOrEmpty(server.workspaceManagerUri)) {
+            throw new IllegalArgumentException("Workspace Manager Service URI cannot be empty");
+        }
+        if (server.testRunnerServiceAccount == null) {
+            throw new IllegalArgumentException("Workspace Manager Service client service account is required");
+        }
+
+        // refresh the client service account token
+        GoogleCredentials serviceAccountCredential =
+                AuthenticationUtils.getServiceAccountCredential(
+                        server.testRunnerServiceAccount, AuthenticationUtils.userLoginScopes);
+        AccessToken accessToken = AuthenticationUtils.getAccessToken(serviceAccountCredential);
+        logger.debug(
+                "Generated access token for workspace manager service client SA: {}",
+                server.testRunnerServiceAccount.name);
+
+        // build the client object
+        ApiClient apiClient = new ApiClient();
+        apiClient.setBasePath(server.workspaceManagerUri);
+        apiClient.setAccessToken(accessToken.getTokenValue());
+
+        return apiClient;
+    }
+
+    /**
+     * Build the Workspace Manager API client object for the given test user and server specifications. This
+     * class maintains a cache of API client objects, and will return the cached object if it already
+     * exists. The token is always refreshed, regardless of whether the API client object was found in
+     * the cache or not.
```

Comment:
Nit: Remove caching from this method comment.

Reply:
Caching comment removed.